### ISSUE TITLE
Rails 4.1.0 troubles?

### ISSUE THREAD

**Issue by AntonTrapp** [anjlab/paranoid2#6]
Some of my tests are failing in 4.1.0, found out that a simple

``` ruby
User.only_deleted
```

leads to:

```
undefined method `default_scoped=' for #<User::ActiveRecord_Relation:0x00000109a6e910>
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/activerecord-4.1.0/lib/active_record/relation/delegation.rb:136:in `method_missing'
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/activerecord-4.1.0/lib/active_record/relation/delegation.rb:99:in `method_missing'
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/paranoid2-1.1.4/lib/paranoid2/scoping.rb:20:in `block in with_deleted'
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/paranoid2-1.1.4/lib/paranoid2/scoping.rb:20:in `tap'
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/paranoid2-1.1.4/lib/paranoid2/scoping.rb:20:in `with_deleted'
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/paranoid2-1.1.4/lib/paranoid2/scoping.rb:16:in `only_deleted'
    from (irb):3
    from /Users/atrapp/.rvm/gems/ruby-2.1.1@creto/gems/railties-4.1.0/lib/rails/commands/console.rb:90:in `start'
```

ruby 2.1.1p76
Rails 4.1.0

default_scoped is really gone, gem should get a major upgrade or a big deprecated flag ;)

**Comment by AntonTrapp** [comment]
I am too stupid to make a pull request (or this feature is buggy). Without many testing I thing that could be the problem and the solution (I don't often use scopes and never used a default_scope):

lib/paranoid2/scoping.rb

``` ruby
def with_deleted
 -        all.tap { |s| s.default_scoped = false }
 +        unscoped
end
```

**Comment by yury** [comment]
Thank you for your feedback.

Just released v1.1.5 with fix for you.
